File: Day41/Java/largestElement.java
public class largestElement {
    static int largestEle(int[] arr,int n){
        int largest = arr[0];
        for(int i=1;i<n;i++){
            if(arr[i]>largest){
                largest = arr[i];
            }
        }
        return largest;
    }
    public static void main(String[] args) {
        int[] arr = {1,3,5,4,6,2,0,9};
        int ans = largestEle(arr, arr.length);
        System.out.println("The largest element from arr is: "+ans);   
    }
}

File: Day41/CPP/largestElement.cpp
#include<bits/stdc++.h>
using namespace std;
int largestElement(vector<int> &arr, int n) {
    int max = arr[0];
    for(int i=1;i<n;i++){
        if(arr[i]>=max) max = arr[i];
    }
    return max;
}
int main()
{
    vector<int> arr = {5, 10, 15, 20, 25};
    int n = arr.size();
    int largest = largestElement(arr, n);
    cout << "Largest element in the array: " << largest << endl;
    return 0;
}